consul.c

```diff
@@ -209,6 +210,7 @@ static void consul_setup() {
 			"id", &ucs->id,
 			"name", &ucs->name,
 			"port", &ucs->port,
+			"address", &ucs->address,
```

Comment:
Just for QA (unless it has a bad meaning) could you please order the parameters alphabetically and by so move this above the "id" ?